etna/collections/models.py

```diff
@@ -543,15 +542,9 @@ class Highlight(Orderable):
         on_delete=models.SET_NULL,
         verbose_name=_("image"),
     )
-    long_description = RichTextField(
-        verbose_name=_("long description"),
-        features=settings.RESTRICTED_RICH_TEXT_FEATURES,
-        max_length=400,
-    )
 
     panels = [
         FieldPanel("image"),
-        FieldPanel("long_description"),
```

Comment:
I'm not sure what the deal is with this being here, but I can't save the page model to get the relations without this being removed.

Reply:
Did you review my branch where I removed it? Maybe some lingering migrations or something

Reply:
I did, but was this removed or just the field? The field does not exist in the DB, I've rebased twice but perhaps this is not in `develop` and I'm using data that's old / modified from that PR.